### PR TITLE
Add navbar , router and Styling with interactive element.

### PR DESCRIPTION
**Pull request by the-faizmohammad**
Description:
This pull request adds interactive elements to the Home component, creates separate Calculator and Quote pages, sets up routing using React Router, and applies basic styling to match the wireframes provided.

**Changes Made:**
1. **Home Component:**
   - Added interactive buttons to navigate to the Calculator and Quote pages.
   - Included relevant styling to improve user experience.

2. **Calculator Component:**
   - Created a new Calculator component to display calculator-related content.
   - Styled the Calculator component to match the wireframe design.

3. **Quote Component:**
   - Created a new Quote component to display inspirational quotes.
   - Styled the Quote component for a consistent look and feel.

4. **React Router Setup:**
   - Configured React Router to handle routing between Home, Calculator, and Quote pages.
   - Added navigation links to all pages for easy navigation.

5. **Styling:**
   - Created separate CSS files for each component and applied styles as per wireframes.

### CODE REVIEW

File: src/spa/home.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            simplifies complex arithmetic and earns popularity for its portability.
+          </p>
+          <p className="paragraph">
+            Electronic calculators emerge, utilizing transistors and integrated circuits to enable
+            and more precise arithmetic calculations.
+          </p>
+        </div>
+      </div>
+    );
+  }
```
Comment (danmainah) [inline src/spa/home.js:92]: - [ ] @the-faizmohammad Great work in adding style to your homepage, however there is a slight difference between your layout and the design given, kindly check the comparison below:


Given Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

File: src/components/Calculator.js [hunk 2]
```diff
@@ -17,20 +17,23 @@ const Calculator = () => {
...
+            className="field"
+            value={calculatorState.next || calculatorState.total || ''}
+            readOnly
+          />
+        </div>
+        <Btn onClick={handleButtonClick} />
       </div>
-      <Btn onClick={handleButtonClick} />
-    </div>
+    </>
```
Comment (danmainah) [inline src/components/Calculator.js]: - [ ] @the-faizmohammad Great work adding stlye to your calculator page, however there are sight differences to the design given, check the comparison below for better understanding:


Given Design          |  Your design     
:-------------------------:|:-------------------------:
image  | image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @the-faizmohammad , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Added a navbar to enable easier navigation between pages.
✔️  Ensured the website has 3 pages Home, Calculator, and Quote.


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] @the-faizmohammad It would have been nice if you added description about the project in your readme file, also add how to install dependancies before running the app.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by DammyShittu — APPROVED** [review]
Hi @the-faizmohammad,

## APPROVED 🟢

## Highlights
✔️ Project met all technical requirements

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — COMMENTED** [review]
# Approved 🙌

Hi @the-faizmohammad 

 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**


## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great
-  ✅ Great work using the right Gitflow 
-  ✅ Your README looks good

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**